### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by mitchellsimoens** [2fd/graphdoc#63]
I see the last commit was 8 months ago but there are PRs that haven't been commented by a maintainer for a year now. I'm afraid to ask, is this project maintained any more? This project is great but there are certain things that could be improved and fixed. Would be sad if this project went inactive.

**Comment by gavinhenderson** [comment]
Second this! Would be a real shame to see this project go unmaintained especially when there are people willing to maintian it.

**Comment by mnovozhylov** [comment]
Same here - would be great to know the actual status

**Comment by gavinhenderson** [comment]
@mitchellsimoens @mnovozhylov - If there isn't a response or we find out the project isn't being maintained would you be interested in maintaining a fork?

**Comment by cabello** [comment]
Yes, I would be up to maintain a fork, I am relying on this project at the moment for several projects.

**Comment by mitchellsimoens** [comment]
I too would be open to maintaining a fork. We need a separate account for it as I don't like these things to be bound to individual accounts and we can still give certain individual accounts certain permissions. Same with a npm org.

**Comment by mitchellsimoens** [comment]
@cabello I did setup a Github [org](https://github.com/GraphiDocsOrg/) and [repo](https://github.com/GraphiDocsOrg/docs). lmk if you want to be added. Need to do some cleanup before publish to npm but I did create a matching org there to grab it. We can always rename things, used graphidocs to be like graphiql but that's my imagination for ya :)

**Comment by gavinhenderson** [comment]
@mitchellsimoens I would like to be part of maintaining the fork too

**Comment by dobrynin** [comment]
Add one more in support of the fork!

**Comment by cabello** [comment]
@mitchellsimoens can you invite me to the org please? I will happily join it.

**Comment by Spolja** [comment]
Any status on the progress of this project, would be a real shame if it stopped completely

**Comment by cabello** [comment]
The project has been forked https://github.com/GraphiDocsOrg/docs and I've been using a patched version that updates graphql dependencies because otherwise documentation would not be generated.

**Comment by 2fd** [comment]
Hi guys, I'm so sorry.
I will resume the development of the project with some changes.
If someone is still interested, they will be welcome.
Also, I'm going to look at the fork to get ideas.

Thanks for everything

**Comment by carldunham** [comment]
So, are we back to inactive again? This repo and the org/fork now?

**Comment by mitchellsimoens** [comment]
@carldunham yea, I stopped because I went in thinking I wasn’t going to be the only one maintaining and turned out I was being only one so when the project I was on changed and no longer needed it, I lost interest. Happy to review any PRs.

**Comment by cabello** [comment]
I thought we would fork and patch it, but the first PR was a big change. It changed the name of the CLI, I already had over 50 repos using the previous cli name, so I did not want the cascade effect of changes causing more changes. Sorry about that, for what is worth, this still generates valid documentation for my use case.

**Comment by carldunham** [comment]
Thanks, all. The work you have done is much appreciated. Sorry it hasn't been able to take off on its own.
